predictsignauxfaibles/utils.py

```diff
@@ -173,3 +176,21 @@ def set_if_not_none(obj, attr, val):
     """
     if val is not None:
         setattr(obj, attr, val)
+
+
+def load_conf(model_name: str = "default"):
+    """
+    Loads a model configuration from a model name
+    Args:
+        model_name: str
+    """
+    conf_filepath = Path("../models") / model_name / "model_conf.py"
+    if not conf_filepath.exists():
+        raise ValueError(f"{conf_filepath} does not exist")
+
+    spec = importlib.util.spec_from_file_location(
+        f"models.{model_name}.model_conf", conf_filepath
+    )
+    model_conf = importlib.util.module_from_spec(spec)
+    spec.loader.exec_module(model_conf)  # pylint: disable=wrong-import-position
+    return model_conf
```

Comment:
perfect 👍🏼 

I think this function already existed in `__main__.py` right ? If so, let's remove it from there to avoir redundancy :)